mvp-spec/json-schemas/post-seed-request.json

```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "post-seed-request",
+    "title": "POST /v1/seed request",
```

Comment:
Should be `POST /paf-proxy/v1/seed request` (missing proxy prefix is confusing, I had to look for implementation to understand)

Reply:
Done